megamek/src/megamek/server/Server.java

```diff
@@ -3722,46 +3731,29 @@ private void determineTurnOrderIUI(IGame.Phase phase) {
         // Now, generate the global order of all teams' turns.
         TurnVectors team_order = TurnOrdered.generateTurnOrder(entities, game);
 
-        // See if there are any loaded units stranded on immobile transports.
-        Iterator<Entity> strandedUnits = game
-                .getSelectedEntities(new EntitySelector() {
-                    public boolean accept(Entity entity) {
-                        if (game.isEntityStranded(entity)) {
-                            return true;
-                        }
-                        return false;
-                    }
-                });
-
         // Now, we collect everything into a single vector.
-        Vector<GameTurn> turns;
-
-        if (strandedUnits.hasNext()
-                && (game.getPhase() == IGame.Phase.PHASE_MOVEMENT)) {
-            // Add a game turn to unload stranded units, if this
-            // is the movement phase.
-            turns = new Vector<GameTurn>(team_order.getTotalTurns()
-                    + team_order.getEvenTurns() + 1);
-            turns.addElement(new GameTurn.UnloadStrandedTurn(strandedUnits));
-        } else {
-            // No stranded units.
-            turns = new Vector<GameTurn>(team_order.getTotalTurns()
-                    + team_order.getEvenTurns());
-        }
-
-        // add the turns (this is easy)
-        while (team_order.hasMoreElements()) {
-            Entity e = (Entity) team_order.nextElement();
-            if (e.isSelectableThisTurn()) {
-                if (!protosMoveMulti && (e instanceof Protomech)
-                        && (e.getUnitNumber() != Entity.NONE)) {
-                    turns.addElement(new GameTurn.UnitNumberTurn(
-                            e.getOwnerId(), e.getUnitNumber()));
-                } else {
-                    turns.addElement(new GameTurn.SpecificEntityTurn(e
-                            .getOwnerId(), e.getId()));
-                }
-
+        Vector<GameTurn> turns = new Vector<GameTurn>(team_order.getTotalTurns()
```

Comment:
This section of code looks identical to that on line 3960. Let's consolidate it into a single method if possible.